pkg/common/constants.go

```diff
@@ -22,6 +22,8 @@ const (
 	LabelAnnotationStorageCapacityPrefix = LabelAnnotationPrefix + "s-"
 	// The dataset annotation
 	LabelAnnotationDataset = LabelAnnotationPrefix + "dataset"
+	// LabelAnnotationDatasetNum indicates the number of the dataset in specific node
+	LabelAnnotationDatasetNum = LabelAnnotationStorageCapacityPrefix + "dataset-num"
```

Comment:
LGTM

Reply:
Why not using LabelAnnotationPrefix + "dataset-num" ？